libPhoneNumber-Demo/libPhoneNumber-DemoTests/libPhoneNumber_DemoTests.swift

```diff
@@ -0,0 +1,34 @@
+//
```

Comment:
Remove the test stuff. I don't think we need it

Reply:
removed tests.